docs/advanced_topics/settings.rst

```diff
@@ -274,6 +274,113 @@ Case-Insensitive Tags
 
 Tags are case-sensitive by default ('music' and 'Music' are treated as distinct tags). In many cases the reverse behaviour is preferable.
 
+Custom User Edit Forms
+----------------------
+
+.. code-block:: python
+
+  WAGTAIL_USER_EDIT_FORM = 'users.forms.CustomUserEditForm'
+
+Allows the default ``UserEditForm`` class to be overridden with a custom form when
+a custom user model is being used and extra fields are required in the user edit form.
+
+.. code-block:: python
+
+  WAGTAIL_USER_CREATION_FORM = 'users.forms.CustomUserCreationForm'
+
+Allows the default ``UserCreationForm`` class to be overridden with a custom form when
+a custom user model is being used and extra fields are required in the user creation form.
+
+.. code-block:: python
+
+  WAGTAIL_USER_CUSTOM_FIELDS = ['country']
+
+A list of the extra custom fields to be appended to the default list.
+
+Custom user forms example
```

Comment:
I think this section really needs to be moved elsewhere, possibly a new page under Advanced Topics -> Customising Wagtail. This page is already a bit out of control - it starts off as a tutorial in how to set up Wagtail from scratch from a barebones Django project, then segues into a Wagtail settings reference. Segueing into a different tutorial is taking things a bit too far :-)